### PR TITLE
Refunds: update copy to reflect 14-day refunds

### PR DESCRIPTION
**Pull request by michaeldcain**
This updates mentions of the 30-day refund policy to the new 14-day refund policy. This change is limited to copy - business logic changes will be made at a future point.

ref: pbOQVh-Lt-p2

**To Test:**
- visual check of changes should suffice
- search codebase for additional mentions of the refund policy

**Screenshots for translation:**
Screen Shot 2021-01-17 at 10 40 59 PM
Screen Shot 2021-01-17 at 10 34 54 PM
Screen Shot 2021-01-17 at 10 33 28 PM
Screen Shot 2021-01-17 at 10 30 59 PM
Screen Shot 2021-01-17 at 10 27 46 PM
Screen Shot 2021-01-17 at 10 21 43 PM

### CODE REVIEW

File: client/landing/gutenboarding/onboarding-block/plans/index.tsx [hunk 1]
```diff
@@ -91,7 +91,7 @@ const PlansStep: React.FunctionComponent< Props > = ( { isModal } ) => {
 				<Title>{ __( 'Select a plan' ) }</Title>
 				<SubTitle>
 					{ __(
-						'Pick a plan that’s right for you. There’s no risk, you can cancel for a full refund within 30 days.'
+						'Pick a plan that’s right for you. There’s no risk, you can cancel for a full refund within 14 days.'
```
Comment (a8ci18n) [inline client/landing/gutenboarding/onboarding-block/plans/index.tsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/me/concierge/shared/appointment-info.js [hunk 2]
```diff
@@ -115,7 +115,7 @@ class AppointmentInfo extends Component {
 							<br />
 							<FormSettingExplanation>
 								{ translate(
-									'Note: You have 30 days from the date of purchase to cancel an unused Quick Start ' +
+									'Note: You have 14 days from the date of purchase to cancel an unused Quick Start ' +
```
Comment (a8ci18n) [inline client/me/concierge/shared/appointment-info.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Note: You have 30 days from the date of purchase to cancel an unused Quick Start session and receive a refund. Please note, if you miss a scheduled session twice, the purchase will be cancelled without a refund.' )` *ES Score: 10*

File: client/my-sites/checkout/composite-checkout/components/concierge-refund-policy.jsx [hunk 3]
```diff
@@ -23,7 +23,7 @@ class ConciergeRefundPolicy extends React.Component {
 
 	renderPolicy = () => {
 		const message = this.props.translate(
-			'You understand that {{refundsSupportPage}}Quick Start Session refunds{{/refundsSupportPage}} are limited to 30 days from the date of purchase.',
+			'You understand that {{refundsSupportPage}}Quick Start Session refunds{{/refundsSupportPage}} are limited to 14 days from the date of purchase.',
```
Comment (a8ci18n) [inline client/my-sites/checkout/composite-checkout/components/concierge-refund-policy.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js [hunk 4]
```diff
@@ -155,11 +155,11 @@ function CheckoutSummaryFeaturesList( props ) {
 
 	let refundText = translate( 'Money back guarantee' );
 	if ( hasDomainsInCart && ! hasPlanInCart ) {
-		refundText = translate( '4 day money back guarantee' );
+		refundText = translate( '4-day money back guarantee' );
```
Comment (a8ci18n) [inline client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js [hunk 5]
```diff
@@ -155,11 +155,11 @@ function CheckoutSummaryFeaturesList( props ) {
...
 	} else if ( hasPlanInCart && ! hasDomainsInCart ) {
 		refundText = hasMonthlyPlan
-			? translate( '7 day money back guarantee' )
-			: translate( '30 day money back guarantee' );
+			? translate( '7-day money back guarantee' )
```
Comment (a8ci18n) [inline client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js [hunk 6]
```diff
@@ -155,11 +155,11 @@ function CheckoutSummaryFeaturesList( props ) {
...
+			: translate( '14-day money back guarantee' );
```
Comment (a8ci18n) [inline client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/index.jsx [hunk 7]
```diff
@@ -223,7 +223,7 @@ export class BusinessPlanUpgradeUpsell extends PureComponent {
 								) }
 							{ ! hasSevenDayRefundPeriod &&
 								translate(
-									'The good news is that you can upgrade your plan today and try the Business plan risk-free thanks to our {{b}}30-day money-back guarantee{{/b}}.',
+									'The good news is that you can upgrade your plan today and try the Business plan risk-free thanks to our {{b}}14-day money-back guarantee{{/b}}.',
```
Comment (a8ci18n) [inline client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/index.jsx [hunk 8]
```diff
@@ -249,7 +249,7 @@ export class BusinessPlanUpgradeUpsell extends PureComponent {
 								) }
 							{ ! hasSevenDayRefundPeriod &&
 								translate(
-									'Simply click the link below and select the Business plan option to upgrade today {{b}}for just {{del}}%(fullPrice)s{{/del}} %(discountPrice)s more{{/b}}. Once you upgrade, you’ll have 30 days to evaluate the plan and decide if it’s right for you.',
+									'Simply click the link below and select the Business plan option to upgrade today {{b}}for just {{del}}%(fullPrice)s{{/del}} %(discountPrice)s more{{/b}}. Once you upgrade, you’ll have 14 days to evaluate the plan and decide if it’s right for you.',
```
Comment (a8ci18n) [inline client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/checkout/upsell-nudge/premium-plan-upgrade-upsell/index.jsx [hunk 9]
```diff
@@ -188,7 +188,7 @@ export class PremiumPlanUpgradeUpsell extends PureComponent {
 						</ul>
 						<p>
 							{ translate(
-								'Try Premium risk-free with our {{strong}}30-day money-back guarantee{{/strong}}. If you don’t love it, you can return to the Personal Plan.',
+								'Try Premium risk-free with our {{strong}}14-day money-back guarantee{{/strong}}. If you don’t love it, you can return to the Personal Plan.',
```
Comment (a8ci18n) [inline client/my-sites/checkout/upsell-nudge/premium-plan-upgrade-upsell/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/plans-features-main/wpcom-faq.jsx [hunk 10]
```diff
@@ -142,7 +142,7 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<FAQItem
 				question={ translate( 'Can I cancel my subscription?' ) }
 				answer={ translate(
-					'Yes. We want you to love everything you do at WordPress.com, so we provide a 30-day' +
+					'Yes. We want you to love everything you do at WordPress.com, so we provide a 14-day' +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/wpcom-faq.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Yes. We want you to love everything you do at WordPress.com, so we provide a 30-day refund on all of our plans. {{a}}Manage purchases{{/a}}.' )` *ES Score: 8*

File: packages/launch/src/focused-launch/plan-details/index.tsx [hunk 11]
```diff
@@ -48,7 +48,7 @@ const PlanDetails: React.FunctionComponent = () => {
 				<Title tagName="h2">{ __( 'Select a plan', __i18n_text_domain__ ) }</Title>
 				<SubTitle tagName="h3">
 					{ __(
-						"There's no risk, you can cancel for a full refund within 30 days.",
+						"There's no risk, you can cancel for a full refund within 14 days.",
```
Comment (a8ci18n) [inline packages/launch/src/focused-launch/plan-details/index.tsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx [hunk 12]
```diff
@@ -115,7 +115,7 @@ const FinalStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep, on
 						{ createInterpolateElement(
 							/* translators: pressing <Link> will redirect user to plan selection step */
 							__(
-								'<Link>Upgrade to Premium</Link> to get access to 13GB storage space, payment collection options, 24/7 Live Chat support, and more. Not sure? Give it a spin—we offer 30-day full-refunds, guaranteed.',
+								'<Link>Upgrade to Premium</Link> to get access to 13GB storage space, payment collection options, 24/7 Live Chat support, and more. Not sure? Give it a spin—we offer 14-day full-refunds, guaranteed.',
```
Comment (a8ci18n) [inline apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx [hunk 13]
```diff
@@ -46,7 +46,7 @@ const PlanStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onN
 					<Title>{ __( 'Select a plan', 'full-site-editing' ) }</Title>
 					<SubTitle>
 						{ __(
-							'Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within 30 days.',
+							'Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within 14 days.',
```
Comment (a8ci18n) [inline apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Review by StefanNieuwenhuis — COMMENTED** [review]
I've checked your changes to Gutenboarding, Step By Step Launch & Focused Launch and they LGTM! 👍

**Comment by michaeldcain** [comment]
Switched back to a draft after some conversation about abstracting the day variable: p1610550051436500-slack-C02AED43D

**Comment by paulbonahora** [comment]
Looking good to me!

Only thing to mention is that at checkout after signup I got it only to read "Money back guarantee" and nothing making reference to the amount of days. I'm not sure of the logic we use to show that vs the amount of days, but I couldn't get a scenario where I got the amount of days mentioned. I'm not intending to change the logic on how we display this, just mentioning that I wasn't able to see 30/14 days in that section. 

Calypso changes looked good to me too.

**Comment by michaeldcain** [comment]
@paulbonahora thanks for the review - can you retest the Checkout summary? I pushed a change that might've fixed it.

File: client/my-sites/plans-features-main/wpcom-faq.jsx [hunk 14]
```diff
@@ -142,8 +142,7 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<FAQItem
 				question={ translate( 'Can I cancel my subscription?' ) }
 				answer={ translate(
-					'Yes. We want you to love everything you do at WordPress.com, so we provide a 30-day' +
-						' refund on all of our plans. {{a}}Manage purchases{{/a}}.',
+					'Yes. We want you to love everything you do at WordPress.com, so we provide a 14-day refund on all of our annual plans and a 7-day refund on all of our monthly plans. {{a}}Manage purchases{{/a}}.',
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/wpcom-faq.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Yes. We want you to love everything you do at WordPress.com, so we provide a 30-day refund on all of our plans. {{a}}Manage purchases{{/a}}.' )` *ES Score: 6*

**Comment by pottedmeat** [comment]
Looks like some of the code uses `purchase.refund_period_in_days` but can't track down in code how this is set.

File: client/landing/gutenboarding/onboarding-block/plans/index.tsx (hunk 1 above)
Comment (nbloomf) [reply client/landing/gutenboarding/onboarding-block/plans/index.tsx]: Is it worth making the number of days a variable with sprintf (or whatever the i18n equivalent is)? That would at least prevent future changes here from requiring another round of translations.

File: client/me/concierge/shared/appointment-info.js (hunk 2 above)
Comment (nbloomf) [reply client/me/concierge/shared/appointment-info.js]: lol

File: client/landing/gutenboarding/onboarding-block/plans/index.tsx (hunk 1 above)
Comment (michaeldcain) [reply client/landing/gutenboarding/onboarding-block/plans/index.tsx]: I actually just dropped that exact commit. 

There's some Slack conversation around when to use the abstracted variable (p1610550051436500-slack-C02AED43D). If I understand correctly, we'd only do this if the number itself is variable in this particular instance (i.e. 7 vs 14, based on plan length, [like it is here](https://github.com/Automattic/wp-calypso/pull/48872/files#diff-010394ebaf934e04dc170d0355b1146369fa4d5db7daa5bb6895eb13cb8219f8R217)). This boils down to there being the possibility of a single string being translated differently in a single language based on the value of that number.

@akirk or @yuliyan might be able to correct me if I'm wrong.

**Comment by michaeldcain** [comment]
> Looks like some of the code uses purchase.refund_period_in_days but can't track down in code how this is set.

@pottedmeat: I believe this comes from the `/purchases/` endpoint. I'll patch that in a separate diff.

**Comment by pottedmeat** [comment]
Looked through code using as many angles as I could and I couldn't find any additional 30-day mentions.

File: client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/index.jsx [hunk 15]
```diff
@@ -213,56 +213,40 @@ export class BusinessPlanUpgradeUpsell extends PureComponent {
...
 						<p>
-							{ hasSevenDayRefundPeriod && // @todo - Remove the duplicate translations once the new string is translated
+							{
+								// Using plural translation because some languages have multiple plural forms and no plural-agnostic.
 								translate(
+									'The good news is that you can upgrade your plan today and try the Business plan risk-free thanks to our {{b}}%(days)d-day money-back guarantee{{/b}}.',
 									'The good news is that you can upgrade your plan today and try the Business plan risk-free thanks to our {{b}}%(days)d-day money-back guarantee{{/b}}.',
 									{
-										args: { days: 7 },
-										components: { b: <b /> },
```
Comment (pottedmeat) [inline client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/index.jsx:221]: I think this `<b />` should still be passed

**Review by pottedmeat — APPROVED** [review]

**Review by pottedmeat — CHANGES_REQUESTED** [review]
I left a note for a minor translation issue.

**Review by pottedmeat — APPROVED** [review]

File: client/landing/gutenboarding/onboarding-block/plans/index.tsx (hunk 1 above)
Comment (yuliyan) [reply client/landing/gutenboarding/onboarding-block/plans/index.tsx]: I don't want to force my opinion, but I think that it's not unlikely that these periods may need to be changed again in the future, so replacing them with variables now would make it easier for future updates with no need for re-translations.

Comment (michaeldcain) [reply client/landing/gutenboarding/onboarding-block/plans/index.tsx]: @yuliyan thanks for weighing in! You're not forcing your opinion - you're the expert. I think I misunderstood the slack thread. I made the change in 7e142fea35de36c9a412280921b5dd4f23109d75 and 8670723182e792f3be04a08a7abad962572d0003.

File: apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx [hunk 16]
```diff
@@ -45,9 +45,13 @@ const PlanStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onN
 				<div>
 					<Title>{ __( 'Select a plan', 'full-site-editing' ) }</Title>
 					<SubTitle>
-						{ __(
-							'Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within 30 days.',
-							'full-site-editing'
+						{ sprintf(
+							/* translators: number of days */
+							__(
+								'Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within %1$d days.',
```
Comment (a8ci18n) [inline apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx:51]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Pick a plan that’s right for you. Switch plans as your needs change. {{br/}} There’s no risk, you can cancel for a full refund within 30 days.' )` *ES Score: 8*

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5443004

Thank you @michaeldcain for including a screenshot in the description! This is really helpful for our translators.

**Review by niranjan-uma-shankar — COMMENTED** [review]
Verified that the Quick Start related pages and the Premium and Business plan upsell pages all display 14-days as the refund period. LGTM within the scope of all places in Calypso that _I know_ of where we mention refunds.

**Review by mattgawarecki — APPROVED** [review]
Everything looks good to me from a Voyager perspective, both code and in the actual browser! :shipit:

**Review by sirbrillig — APPROVED** [review]
Looks good to me from a code perspective!

**Comment by DavidRothstein** [comment]
>Looks like some of the code uses `purchase.refund_period_in_days`

FYI, I found one place where this affects the UI text, and therefore isn't fixed by this pull request: `client/me/purchases/cancel-purchase/refund-information.jsx` (see the usages of `refundPeriodInDays` in that file).

You can reproduce by going to the Manage Purchase page for a newly-purchased subscription and then clicking "Cancel Subscription and Refund":

Screenshot from 2021-01-21 16-00-01

**Comment by rcanepa** [comment]
@robertbpugh requested a slightly different copy update for the FAQ section located on the Jetpack Plans/Pricing page. Should we push a commit to this PR or open a different one once this gets merged?

Here is Rob's request: p1611271360043700-slack-CQXNTE9K9.

**Comment by michaeldcain** [comment]
@rcanepa @robertbpugh: since that change will require re-translation, I think it might be better to make a separate PR. If you do and add the `string freeze` label before Monday, it'll go out with the next batch of translation strings.

**Comment by michaeldcain** [comment]
Addressing @DavidRothstein's comment in D55866-code.

**Comment by michaeldcain** [comment]
Addressing @rcanepa comment (and @robertbpugh's request) in https://github.com/Automattic/wp-calypso/pull/49201

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5443004).

**Pull request merged**